core/partial-match-registry/src/index.ts

```diff
@@ -75,4 +75,9 @@ export class Registry<V> {
   clear() {
     this.store = createSortedArray<V>();
   }
+
+  /** Check if the registry is empty*/
+  isRegistryEmpty() {
```

Comment:
@sugarmanz @hborawski Do we expose the partial match registry on every platform(It doesn't look like we do but I just wanted to check)? If so we should also add this to every platforms API.

Reply:
https://github.com/player-ui/player/blob/bazel-6/ios/core/Sources/CorePlugins/PartialMatchFingerprintPlugin.swift#L75

We do have it in iOS, but it's not exposed to users + no Swift API currently